Firestore/core/src/core/filter.h

```diff
@@ -19,14 +19,17 @@
 
 #include <iosfwd>
 #include <memory>
+#include <mutex>  // NOLINT(build/c++11)
```

Comment:
Do I need this here? 🤔

Reply:
Probably not anymore. It was there for std::once_flag.